mythril/laser/ethereum/cfg.py

```diff
@@ -55,11 +66,7 @@ def __init__(
         self.function_name = function_name
         self.flags = NodeFlags()
 
-        # Self-assign a unique ID
-        global gbl_next_uid
-
-        self.uid = gbl_next_uid
-        gbl_next_uid += 1
+        self.uid = new_node_uid()
```

Comment:
what is this used for? could we just use `hash(self)`?

Reply:
Sequential node id was useful to read statespace in ancient times(now statespace is unreadable because storage and symbolic balances), So we can replace it with hash.